docs/pr_review_process.md

```diff
@@ -0,0 +1,71 @@
+2. Find appropriate sample data (either already in the shared directory or new data), ideally not data used by the PR author.
+3. Update your local development environment to use the PR branch.
+4. Create a new Jupyter notebook named `pr<num>_<short description>.ipynb`
+
+### Step 2: Review the PR functionality
+
+Read the PR documentation and use your intuition to run relevant parts of PlantCV to test the PR code. Potential findings to report to the PR author and/or to propose fixes for:
+
+1. **Errors**: the code does not work as written or does not function as intended (on new data for example).
+2. **Process**: in using the code, you think the process can be simplified or made clearer (e.g., fewer inputs, renamed arguments, etc.).
+3. **Documentation**: both the documentation and docstrings should match the code in terms of syntax and process.
+
+Iterate as needed.
+
+### Step 3: Review the tests
+
+1. Do all tests pass?
+    1. Running relevant tests locally: `pytest -v tests/tests.py -k test_name_or_keywords`
+2. Does coverage remain at 100%?
+3. Do any new or modified tests accurately test the PR code?
+
+Propose updates as needed.
+
+### Step 4: Test the documentation
+
+Reviewing the documentation code is important, but compiling the documentation locally ensures that the resulting pages are rendered correctly. The easiest way to view the documentation live on your local machine is to run: `mkdocs serve --theme readthedocs`
+
+Some common oversights that are hard to see in the code but easy to detect in the compiled documentation:
+
+1. New pages are not added to the table of contents plantcv/mkdocs.yml (mkdocs will display a warning when you run the command above).
+2. Typos in page or image filenames (mkdocs will often display a warning, or the image may appear as a broken link).
+3. Typos in formatting markup lead to incorrectly displayed styling (e.g., not closing bold or italics, not having a blank line before a bulleted or numbered list, etc.).
+4. Changes to function input/output signatures or new functions should be added to updating.md.
+
+### Step 5: Review the code
+
+Once the PR is working as intended, review the code itself for potential improvements. Some common types of improvements to suggest:
+
+1. **Uniform style (linting)**: Most IDEs, including PyCharm and VScode, will indicate when the code does not match the community style guides laid out in PEP8. The command-line tool `flake8` can also be used if an IDE that supports these checks cannot be used.
+2. **Code simplification**: Any reorganization that makes the code more succinct, readable, maintainable, etc. In particular, complex, nested logical blocks and for loops increase code complexity (and testing demand).
+3. **Algorithmic improvement**: Could potentially be an alternate approach that is significantly faster or more user-friendly.
+4. **Scope**: Check any new code and the tests.py module (no need to fix out-of-scope linting issues).
```

Comment:
what do you mean by the first sentence?

Reply:
Instead of < Check any new code and the tests.py module (no need to fix out-of-scope linting issues). > update to < Check any new code and test data in the `plantcv/tests module (no need to fix out-of-scope linting issues). >